### PR TITLE
6123 get api actor

### PR DESCRIPTION
**Pull request by dinhquang251**
get Data from API after show ActorFragment
https://edu-redmine.framgia.vn/issues/6123

### CODE REVIEW

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_35/screen/actor/fragment/MoviesFragment.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        initData();
+        setAdapter();
+    }
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/actor/fragment/MoviesFragment.java]: remove bl

File: app/src/main/res/layout/activity_actor.xml [hunk 2]
```diff
@@ -78,10 +85,12 @@
                 app:tabSelectedTextColor="?android:attr/textColorPrimaryInverse" />
         </android.support.design.widget.AppBarLayout>
         ​
+
         <android.support.v4.view.ViewPager
             android:id="@+id/viewpager_actor"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_actor.xml]:
```suggestion
            android:id="@+id/view_pager_actor"
```

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/ProductionAdapter.java [hunk 3]
```diff
@@ -63,4 +66,8 @@ private void bindData(Company company) {
             mItemProductionBinding.executePendingBindings();
         }
     }
+
+    public interface onItemClicked {
+        void produceClickListener();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/ProductionAdapter.java:71]: wrong name! rename it

File: app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/CharacterAdapter.java [hunk 4]
```diff
@@ -48,17 +50,24 @@ public void replaceData(List<Actor> actors) {
...
 
         private void bindData(Actor actor) {
             mItemDetailViewModel.setActor(actor);
             mItemActorBinding.executePendingBindings();
+            mItemDetailViewModel.setActorKey(String.valueOf(actor.getId()));
         }
     }
+
+    public interface onItemClicked {
+        void actorClickListener(String key);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/CharacterAdapter.java:71]: wrong name! rename it

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/ProductionAdapter.java [hunk 5]
```diff
@@ -63,4 +66,8 @@ private void bindData(Company company) {
             mItemProductionBinding.executePendingBindings();
         }
     }
+
+    public interface OnItemClickListener {
+        void produceClickListener();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/ProductionAdapter.java:71]:
```suggestion
        void clickProducerListener();
```

File: app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/CharacterAdapter.java [hunk 6]
```diff
@@ -48,17 +50,24 @@ public void replaceData(List<Actor> actors) {
...
 
         private void bindData(Actor actor) {
             mItemDetailViewModel.setActor(actor);
             mItemActorBinding.executePendingBindings();
+            mItemDetailViewModel.setActorKey(String.valueOf(actor.getId()));
         }
     }
+
+    public interface OnItemClickListener {
+        void actorClickListener(String key);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/CharacterAdapter.java:71]:
```suggestion
        void clickActorListener(String key);
```

Comment (NguyenMinhTuanB) [reply app/src/main/java/com/framgia/moviedb_35/screen/detail/adapter/CharacterAdapter.java:71]: tương tự ở những medthod khác

**Pull request merged**